openghg/standardise/footprint/_acrg_org.py

```diff
@@ -0,0 +1,162 @@
+import logging
+from pathlib import Path
+from collections import defaultdict
+from typing import DefaultDict, Dict, Literal, List, Optional, Union
+from xarray import Dataset
+
+from openghg.util import species_lifetime, timestamp_now, check_function_open_nc
+from openghg.store import infer_date_range, update_zero_dim
+from openghg.types import multiPathType
+
+logger = logging.getLogger("openghg.standardise.footprint")
+logger.setLevel(logging.DEBUG)  # Have to set level for logger as well as handler
+
+
+def parse_acrg_org(
+    filepath: multiPathType,
+    site: str,
+    domain: str,
+    model: str,
+    inlet: str,
+    species: str,
+    metmodel: Optional[str] = None,
+    network: Optional[str] = None,
+    period: Optional[Union[str, tuple]] = None,
+    continuous: bool = True,
+    high_spatial_resolution: bool = False,
+    high_time_resolution: bool = False,
+    short_lifetime: bool = False,
+    chunks: Optional[Dict] = None,
+) -> Dict:
+    """
+    Read and parse input emissions data in original ACRG format.
+
+    Args:
+        filepath: Path of file to load
+        site: Site name
+        domain: Domain of footprints
+        model: Model used to create footprint (e.g. NAME or FLEXPART)
+        inlet: Height above ground level in metres. Format 'NUMUNIT' e.g. "10m"
+        metmodel: Underlying meteorlogical model used (e.g. UKV)
+        species: Species name. For a long-lived species this should be "inert".
+        network: Network name
+        period: Period of measurements. Only needed if this can not be inferred from the time coords
+        continuous: Whether time stamps have to be continuous.
+        high_spatial_resolution : Indicate footprints include both a low and high spatial resolution.
+        high_time_resolution: Indicate footprints are high time resolution (include H_back dimension)
+                        Note this will be set to True automatically if species="co2" (Carbon Dioxide).
+        short_lifetime: Indicate footprint is for a short-lived species. Needs species input.
+                        Note this will be set to True if species has an associated lifetime.
+        chunks: Chunk size to use when parsing NetCDF, useful for large datasets.
```

Comment:
```suggestion
       chunks: Chunk schema to use when storing data the NetCDF. It expects a dictionary of dimension name and chunk size, for example {"time": 100}. If None then a chunking schema will be set automatically by OpenGHG.
```

Reply:
Ok I added those changes locally and committed.